src/index.html

```diff
@@ -2,13 +2,76 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta
+      name="viewport"
+      content="width=device-width,
+      user-scalable=no,
+      initial-scale=1.0,
+      maximum-scale=1.0,
+      minimum-scale=1.0"
+    >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+
+    <header class="header">
+
+      <a href="#home">
+        <img
+          src="./images/logo.png"
+          alt="moyoLogo"
+          class="header__image"
+        >
+      </a>
+
+      <nav class="nav">
+
+        <ul class="nav__list">
+
+          <li class="item">
+            <a href="#" class="link is-active">Apple</a>
+          </li>
+
+          <li class="item">
+            <a href="#" class="link">Samsung</a>
+          </li>
+
+          <li class="item">
+            <a href="#" class="link">Smartphones</a>
+          </li>
+
+          <li class="item">
+            <a
+              href="#"
+              class="link"
+              data-qa="hover"
+            >Laptops & Computers
+            </a>
+          </li>
+
+          <li class="item">
+            <a href="#" class="link">Gadgets</a>
+          </li>
+
+          <li class="item">
+            <a href="#" class="link">Tablets</a>
+          </li>
+
+          <li class="item">
+            <a href="#" class="link">Photo</a>
+          </li>
+
+          <li class="item">
+            <a href="#" class="link">Video</a>
+          </li>
+
+        </ul>
+
+      </nav>
+
+    </header>
+
```

Comment:
```suggestion
        </ul>
      </nav>
    </header>
```
empty lines between parent and child elements are redundant